controllers/account.js

```diff
@@ -24,23 +24,44 @@ module.exports.renderChangeProfile = async (req, res) => {
     defaultProfileImg,
   });
 };
+// module.exports.updateProfile = async (req, res) => {
```

Comment:
Why is this commented/here at all?

Reply:
it was the old `updateProfile` i kept as a reference before making sure the new thing works correctly. Simply forgot to remove it, will fix that.